src/main/java/org/mockito/MockSettings.java

```diff
@@ -249,10 +249,10 @@
      * @since 1.10.12
```

Comment:
Please add documentation (e.g. no parameters means that Mockito will look for parameter-less constructor, etc.) and change the @since tag.